data/human/human missions.txt

```diff
@@ -5567,6 +5567,79 @@ mission "Quicksilver Mixup 1"
 event "quicksilver mixup 2 available"
 
 
+mission "Quicksilver Mixup 2"
+	name "Misposted Mail Delivery"
+	description "Deliver <tons> of misposted mail from <origin> to <planet> by <date> and then come back with the return shipment. You will be paid <payment> after you return from <planet>."
+	minor
+	deadline 10 1
+	source "Silver"
+	destination "Quicksilver"
+	cargo "misposted mail" 38
+	"apparent payment" 100000
+	to offer
+		has "Quicksilver Mixup 1: done"
+		has "quicksilver mixup 2 available"
```

Comment:
```suggestion
		has "event: quicksilver mixup 2 available"
```
Caught this while testing. Need that `event` bit in front to use this as a condition.

Reply:
Nice, thanks!